language/de/install.php

```diff
@@ -220,371 +82,136 @@
-	'VERSION_UP_TO_DATE'			=> 'Deine phpBB-Installation ist auf dem neuesten Stand. Obwohl keine Updates zur Verfügung stehen, kannst du fortfahren, um die Dateien auf Gültigkeit zu überprüfen.',
-	'VERSION_UP_TO_DATE_ACP'		=> 'Deine phpBB-Installation ist auf dem neuesten Stand. Es stehen keine Updates zur Verfügung.',
-	'VIEWING_FILE_CONTENTS'			=> 'Dateiinhalte anzeigen',
-	'VIEWING_FILE_DIFF'				=> 'Dateiunterschiede anzeigen',
-
-	'WRONG_INFO_FILE_FORMAT'	=> 'Ungültiges Dateiformat',
+	//
+	// Admin data
+	//
+	'STAGE_ADMINISTRATOR'	=> 'Administrator-Details',
+
+	// Form labels
+	'ADMIN_CONFIG'				=> 'Administrator-Konfiguration',
+	'ADMIN_PASSWORD'			=> 'Administrator-Passwort',
+	'ADMIN_PASSWORD_CONFIRM'	=> 'Bestätigung des Administrator-Passworts',
+	'ADMIN_PASSWORD_EXPLAIN'	=> 'Bitte gib ein Passwort mit einer Länge von 6 bis 30 Zeichen ein.',
+	'ADMIN_USERNAME'			=> 'Benutzername des Administrators',
+	'ADMIN_USERNAME_EXPLAIN'	=> 'Bitte gib einen Benutzernamen mit einer Länge von 3 bis 20 Zeichen ein.',
+
+	// Errors
+	'INST_ERR_EMAIL_INVALID'		=> 'Die angegebene E-Mail-Adresse ist ungültig.',
+	'INST_ERR_PASSWORD_MISMATCH'	=> 'Die eingegebenen Passwörter stimmen nicht überein.',
+	'INST_ERR_PASSWORD_TOO_LONG'	=> 'Das eingegebene Passwort ist zu lang. Die maximale Länge beträgt 30 Zeichen.',
+	'INST_ERR_PASSWORD_TOO_SHORT'	=> 'Das eingegebene Passwort ist zu kurz. Die minimale Länge beträgt 6 Zeichen.',
+	'INST_ERR_USER_TOO_LONG'		=> 'Der von dir angegebene Benutzername ist zu lang. Die maximale Länge beträgt 20 Zeichen.',
+	'INST_ERR_USER_TOO_SHORT'		=> 'Der von dir angegebene Benutzername ist zu kurz. Die minimale Länge beträgt 3 Zeichen.',
+
+	//
+	// Board data
+	//
+	// Form labels
+	'BOARD_CONFIG'		=> 'Konfiguration des Boards',
+	'DEFAULT_LANGUAGE'	=> 'Standard-Sprache',
+	'BOARD_NAME'		=> 'Titel des Boards',
+	'BOARD_DESCRIPTION'	=> 'Kurze Beschreibung des Boards',
+
+	//
+	// Database data
+	//
+	'STAGE_DATABASE'	=> 'Datenbank-Einstellungen',
+
+	// Form labels
+	'DB_CONFIG'				=> 'Datenbank-Konfiguration',
+	'DBMS'					=> 'Datenbank-Typ',
+	'DB_HOST'				=> 'Datenbankserver-Hostname oder DSN',
+	'DB_HOST_EXPLAIN'		=> 'DSN steht für Data Source Name und ist nur für ODBC-Installationen relevant. Bei PostgreSQL wird mit <em>localhost</em> eine Verbindung zum lokalen Server über UNIX-Domain-Socket hergestellt und mit <em>127.0.0.1</em> über TCP. Für SQLite ist der vollständige Pfad der Datenbank-Datei anzugeben.',
+	'DB_PORT'				=> 'Datenbankserver-Port',
+	'DB_PORT_EXPLAIN'		=> 'Lass dieses Feld frei, es sei denn, du weißt, dass der Server nicht den Standard-Port verwendet.',
+	'DB_PASSWORD'			=> 'Datenbank-Passwort',
+	'DB_NAME'				=> 'Name der Datenbank',
+	'DB_USERNAME'			=> 'Datenbank-Benutzername',
+	'TABLE_PREFIX'			=> 'Präfix der Tabellen in der Datenbank',
+	'TABLE_PREFIX_EXPLAIN'	=> 'Der Tabellen-Präfix muss mit einem Buchstaben beginnen und darf nur Buchstaben, Ziffern und Unterstriche enthalten.',
```

Comment:
In Zeile 196 taucht das Präfix wieder auf. Hier finde ich die Übersetzung auch etwas unverständlich. 
'INST_ERR_PREFIX'				=> 'Es existieren bereits Tabellen mit dem angegebenen Präfix, bitte wähle einen alternativen.',

Wäre "bitte wähle ein anderes." nicht verständlicher?